packages/flutter_tools/lib/src/ios/code_signing.dart

```diff
@@ -14,34 +14,57 @@ import '../base/process.dart';
 import '../base/terminal.dart';
 import '../globals.dart';
 
+/// When there are no development certificates at all. The user likely never
+/// did any iOS development.
```

Comment:
Here and below: prefer starting const doc comments with a noun phrase.

https://www.dartlang.org/guides/language/effective-dart/documentation#prefer-starting-variable-getter-or-setter-comments-with-noun-phrases